jupyterkernel/cadabra2_jupyter/kernel.py

```diff
@@ -1,21 +1,25 @@
 import ipykernel.kernelbase
 import sys
 import traceback
+import re
+import string
 
 import cadabra2
-from cadabra2_jupyter.context import _exec_in_context, _attatch_kernel_server
+from cadabra2_jupyter.context import SandboxContext
 from cadabra2_jupyter.server import Server
 from cadabra2_jupyter import __version__
 
+_accepted_chars = string.ascii_letters
+
 
 class CadabraJupyterKernel(ipykernel.kernelbase.Kernel):
     implementation = "cadabra_kernel"
     implementation_version = __version__
     language_info = {
         "name": "cadabra2",
-        "codemirror_mode": "python",
-        "pygments_lexer": "python",
-        "mimetype": "text/python",
+        "codemirror_mode": "cadabra2",
+        "pygments_lexer": "cadabra2",
```

Comment:
These also should be changed back to `"python"` really, so that the Python syntax highlights still work. Alternatively, I can commit a tentative CodeMirror file for Cadabra, so that the use of `#` as a wildcard and comments is at least consistent.